### PR TITLE
Translate README to Chinese.

### PR DESCRIPTION
**Pull request by dingjs**
Translate README to Chinese.

### CODE REVIEW

**Comment by deekoder** [comment]
Awaiting yihuaf here.

**Comment by dingjs** [comment]
ping @yihuaf 
Thanks for you effort,These are the last PRs of the translation.

File: README_zh_CN.md [hunk 1]
```diff
@@ -0,0 +1,157 @@
+# Go Music Player App [![Gitter](https://badges.gitter.im/Join%20Chat.svg)](https://gitter.im/minio/minio?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge)
+
+![minio_GO1](https://github.com/minio/minio-go-media-player/blob/master/docs/screenshots/minio-go1.jpg?raw=true)
+
+本示例将会手把手（限女生）指导你如何用Golang构建一个简单的音乐播放器。在这个app中，我们会向你展示如何从Minio Server上获取你的音频文件。你可以通过[这里](https://github.com/minio/minio-go-media-player)获取完整的代码，代码是以Apache 2.0 License发布的。
```
Comment (yihuaf) [inline README_zh_CN.md:5]: 哈哈哈。。。

Comment (yihuaf) [inline README_zh_CN.md:5]: Apache 2.0 License -> 协议？ as long as it's consistant

**Review by yihuaf — APPROVED** [review]

File: README_zh_CN.md (hunk 1 above)
Comment (dingjs) [reply README_zh_CN.md:5]: It's consistant.We use License instead of 协议

Comment (dingjs) [reply README_zh_CN.md:5]: 留着不？

Comment (yihuaf) [reply README_zh_CN.md:5]: 留着吧 ：）

Comment (yihuaf) [reply README_zh_CN.md:5]: Sounds good to me.

**Review by yihuaf — APPROVED** [review]

**Comment by yihuaf** [comment]
Good to go.

**Pull request merged**